### ISSUE TITLE
oracle_sysconfig variable

### ISSUE THREAD

**Issue by barbudone** [oravirt/ansible-oracle#15]
```
$ pwd
ansible-oracle
$ grep -R oracle_sysconfig *
orahost/defaults/main.yml:  oracle_sysconfig:
orahost/tasks/main.yml:    with_items: oracle_sysconfig
```

IMHO, that names of the variable *_sysconfig, which indeed is used by the module sysctl,
and the values finally go to /etc/sysctl.conf, not something in /etc/sysconfig/ dir.

So, my opinion+question is, this variable needs to be called like: oracle_sysctl :+1:

**Comment by oravirt** [comment]
Sure.
The change has the possibility to break things if the variable is defined elsewhere (group_vars/host_vars) but I guess it's easy enough to change that.

**Comment by oravirt** [comment]
Fixed with be6b2de
